config/routes.rb

```diff
@@ -1,9 +1,8 @@
 Rails.application.routes.draw do
   resources :people
-  resources :events, only: [:show, :create, :new, :upcoming] do
-    get "/upcoming-events", to: 'events#upcoming', on: :collection
-  end
+  resources :events, only: [:show, :create, :new, :upcoming]
   resources :presentations, only: [:show, :create, :new]
 
   root 'welcome#index'
+  get "/upcoming-events", to: 'events#upcoming'
```

Comment:
For me purely from an aesthetic point of view I like it the way it is. The shorter a URL is for me the better but I think as the application grows it might be nice to have it nested as our config file could become a mess and it might just be a bit easier to find for future devs this way. I am like 65% nested and 35% the way it is.

Writing out that comment I think I am going to change it.